tests/config/baseTest.ts

```diff
@@ -57,33 +62,48 @@ class DriverMode {
 }
 
 class ServiceMode {
-  private _playwrightObejct: any;
+  private _playwrightObject: any;
   private _client: any;
   private _serviceProcess: childProcess.ChildProcess;
+  private _tmpConfigFile?: string;
 
-  async setup(workerIndex: number) {
-    const port = 10507 + workerIndex;
-    this._serviceProcess = childProcess.fork(path.join(__dirname, '..', '..', 'lib', 'cli', 'cli.js'), ['run-server', String(port)], {
-      stdio: 'pipe'
+  async setup(workerIndex: number, options?: PlaywrightModeSetupOptions) {
+    if (options?.acceptForwardedPorts) {
+      this._tmpConfigFile = path.join(os.tmpdir(), `pw-server-config-${workerIndex}.json`);
```

Comment:
I'd prefer explicit CLI arguments for now, e.g. "first arg is a port number", "other args are forwarded ports" or something like that. Or maybe a JSON string argument for the ports. But not the temp file.

Reply:
I just wanted to align it with the implementation of our launch-server mode.